README.md

```diff
@@ -70,6 +70,13 @@ The directives have the following options:
     Button()
 ```
 
+### Configuration
+
+You conf.py has two extra configuration options:
+
+ * jupyter_sphinx_require_url: url for `require.js` (if your theme already provides this, set it to False or '')
```

Comment:
How do I say that I want to *not* use require (i.e., my theme doesn't provide it, and I don't want it on the page)